src/Symfony/Component/DependencyInjection/Loader/YamlFileLoader.php

```diff
@@ -81,10 +81,6 @@ class YamlFileLoader extends FileLoader
         'shared' => 'shared',
         'lazy' => 'lazy',
         'public' => 'public',
-        'abstract' => 'abstract',
-        'deprecated' => 'deprecated',
-        'factory' => 'factory',
-        'arguments' => 'arguments',
```

Comment:
Mostly, I thought these were an "abuse" of instanceof - these things are better handled with a normal parent-child. But, I suppose there's nothing wrong with having them.

However, I think `arguments` should not be included. A definition could have multiple instanceof. And with parent-child, the child opts into its 1 specific parent. But with instanceof, these can be applied without the child knowing. I feel like there will be merging edge-cases with arguments.